tests/utils/validator_manager.go

```diff
@@ -1743,58 +1767,8 @@ func PackInitialValidator(iv interface{}) ([]byte, error) {
 func PChainProposerVMWorkaround(
 	pchainWallet pwallet.Wallet,
 ) {
-	// Workaround current block map rules
-	destAddr, err := address.ParseToID(DefaultPChainAddress)
-	Expect(err).Should(BeNil())
 	log.Println("Waiting for P-Chain...")
 	time.Sleep(30 * time.Second)
-
```

Comment:
🤩  love that this is getting deleted. Ideally we can also replace the sleep here eventually..